### PR TITLE
Version/3.19

### PR DESCRIPTION
**Pull request by MrAnno**

### CODE REVIEW

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by kira-syslogng** [comment]
Build SUCCESS

File: NEWS.md [hunk 1]
```diff
@@ -1,86 +1,64 @@
...
- * Add support for whitespace in template functions (#2271)
- * verify the configuration is the same as on the disk (#2345)
+ * HTTP load balancer (#2347)
+ * Slack destination (#2451)
+ * Add Cisco Catalyst formatted triplets support to cisco-parser() (#2394)
+ * Add RFC5424 syslog support to the system() source (FreeBSD 12.0 support) (#2430)
+ * Add network `interface()` option to network sources (#2389)
+ * Add so-reuseport() to network drivers (#2379)
+ * Enable supporting HTTP redirects (#2136)
+ * Configurable text separator for xml() parser (#2459)
```
Comment (gaborznagy) [inline NEWS.md]: We just discussed this IRL and PR #2459 should not be merged just before OSE release 3.19.

Comment (furiel) [reply NEWS.md]: @MrAnno I force-pushed out this line.

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Pull request merged**